### PR TITLE
Convert to axios move xray

### PR DESCRIPTION
**Pull request by tirianmal**
I've incorporated Ivan's changes plus converted to axios. It passes existing tests.

### CODE REVIEW

File: src/fulfiller_identity_proxy.js [hunk 1]
```diff
@@ -26,25 +25,30 @@ class FulfillerIdentityProxy {
     return new Promise((resolve, reject) => {
       self.xray.captureAsyncFunc('FulfillerIdentity.getFulfillerById', function (subsegment) {
         self.authenticator.getAuthorization().then((authorization) => {
+          const axiosInstance = axios.create({
```
Comment (istanishev) [inline src/fulfiller_identity_proxy.js]: If we are going to create an instance, can we "cache" it by creating it globally? Do we need a dedicated instance or we can just use the default one?

**Review by istanishev — APPROVED** [review]

File: src/fulfiller_identity_proxy.js (hunk 1 above)
Comment (tirianmal) [reply src/fulfiller_identity_proxy.js]: we don't need a dedicated instance and can just use the default.

**Comment by tirianmal** [comment]
I will note that we don't have a test of the POST (or the PUT) interface. Thoughts?

**Review by indirap — APPROVED** [review]

**Review by rnowosielski — APPROVED** [review]
Looks good.

**Comment by tirianmal** [comment]
ok, so I changed it from creating a new axios instance to using the default. 

Someone with write access will need to merge the PR if/when they feel it is ready.

**Pull request merged**